src/test/java/co/cask/hydrator/KafkaBatchTest.java

```diff
@@ -181,7 +184,7 @@ public void testKafkaSource() throws Exception {
     messages.put("e", "5,dwayne,johnson");
     sendKafkaMessage("users", messages);
     workflowManager.start();
-    TimeUnit.SECONDS.sleep(10);
+    TimeUnit.SECONDS.sleep(30);
```

Comment:
I think you can remove this and change next line to `workflowManager.waitForRuns(ProgramRunStatus.COMPLETED, 2, 1, TimeUnit.MINUTES);`